src/vue/forms/TransferForm.vue

```diff
@@ -91,32 +93,34 @@
               <textarea-field
                 name="transfer-description"
                 v-model="form.subject"
+                @input="former.setAttr('subject', form.subject)"
                 :label="'transfer-form.subject-lbl' | globalize({
                   length: 250
                 })"
                 :maxlength="250"
-                :readonly="view.mode === VIEW_MODES.confirm"
+                :readonly="formMixin.isDisabled"
                 :disabled="!balance"
               />
             </div>
           </div>
 
           <div
             class="transfer__fee-box"
-            v-if="isFeesLoaded && view.mode === VIEW_MODES.confirm &&
+            v-if="isFeesLoaded && formMixin.isConfirmationShown &&
               fees.isAny
             "
           >
             <fees-renderer
               :fees-collection="fees"
-              :paid-for-destination.sync="form.isPaidForRecipient"
+              :asset-code="form.asset.code"
```

Comment:
why do you add this prop? Do you use this prop in component fees-renderer? Does this component fees-renderer have this prop?